packages/autocomplete-plugin-algolia-insights/src/createClickedEvent.ts

```diff
@@ -0,0 +1,21 @@
+import { AlgoliaInsightsHit, ClickedObjectIDsAfterSearchParams } from './types';
+
+type CreateClickedEventParams = {
+  item: AlgoliaInsightsHit;
+  items: AlgoliaInsightsHit[];
+};
+
+export function createClickedEvent({
+  item,
+  items,
+}: CreateClickedEventParams): Omit<
+  ClickedObjectIDsAfterSearchParams,
+  'eventName'
+> {
+  return {
+    index: item.__autocomplete_indexName,
+    objectIDs: [item.objectID],
+    positions: [1 + items.findIndex((x) => x.objectID === item.objectID)],
```

Comment:
are we keeping track somewhere which polyfills are required? Maybe a similar approach as https://github.com/algolia/instantsearch.js/pull/4584 but outputting a list of polyfills could work?

Reply:
No, although we don't want to provide support for IE.

Having a similar approach would be great. Would you be interested in implementing it?